src/imguiclient/imguiapp.cpp

```diff
@@ -2380,6 +2380,36 @@ void ImGuiApp::FileManagerDescriptorPdbConfig(ProgramFileDescriptor &descriptor)
     }
 }
 
+void ImGuiApp::ShowRemoveFileConfirmationPopup(bool &erased)
+{
+    // Center the modal popup
```

Comment:
I think you can remove all comments in this function because code is self-explanatory.